packages/analytics/analytics-chart/src/utils/colors.ts

```diff
@@ -51,3 +71,50 @@ export const darkenColor = (hex: string, amt: number): string => {
 }
 
 export const accessibleGrey = '#6f7787' // grey-500
+
+export const apiRequestStatusCodeColors = {
+  100: { background: '#f0f5fd', text: '#10599e' },
+  200: { background: '#e5f7f4', text: '#1b6955' },
+  300: { background: '#fff1d5', text: '#a06027' },
+  400: { background: '#fee3b6', text: '#944d19' },
+  500: { background: '#fedada', text: '#852021' },
+}
+
+export const statusCodeBadgeBackgroundColor = (statusCode: string) => {
+  // @ts-ignore TODO
+  return apiRequestStatusCodeColors[Math.floor(statusCode / 100) * 100]?.background || '#fafafa'
+}
+
+// TODO: Is this unused ?
```

Comment:
Weird, it looks like it is.  I think it's better to remove it than have it commented out.